### ISSUE TITLE
Linker error with MSVC while building a project which uses liblo

### ISSUE THREAD

**Issue by Rossmaxx** [radarsat1/liblo#145]
I've been trying to build rtosc, a project which uses liblo as one of it's deps. I am working with the lead dev who is working on porting rtosc to msvc. Don't know if it's related here but anyway, here's the failure logs.

<details> 
<summary>build failure logs</summary>
  Building Custom Rule D:/BuildAndGit/zynaddsubfx/rtosc/CMakeLists.txt
  liblo.c
LINK : warning LNK4044: unrecognized option '/LC:/vcpkg/installed/x64-windows/debug/lib'; ignored [D:\BuildAndGit\zynaddsubfx\rtosc\build\liblo.vcxproj]
LINK : warning LNK4044: unrecognized option '/llo'; ignored [D:\BuildAndGit\zynaddsubfx\rtosc\build\liblo.vcxproj]
liblo.obj : error LNK2019: unresolved external symbol lo_message_new referenced in function speed_liblo_write [D:\BuildAndGit\zynaddsubfx\rtosc\build\liblo.vcxproj]
liblo.obj : error LNK2019: unresolved external symbol lo_message_free referenced in function speed_liblo_write [D:\BuildAndGit\zynaddsubfx\rtosc\build\liblo.vcxproj]
liblo.obj : error LNK2019: unresolved external symbol lo_message_add_int32 referenced in function speed_liblo_write [D:\BuildAndGit\zynaddsubfx\rtosc\build\liblo.vcxproj]
liblo.obj : error LNK2019: unresolved external symbol lo_message_add_float referenced in function speed_liblo_write [D:\BuildAndGit\zynaddsubfx\rtosc\build\liblo.vcxproj]
liblo.obj : error LNK2019: unresolved external symbol lo_message_add_string referenced in function speed_liblo_write [D:\BuildAndGit\zynaddsubfx\rtosc\build\liblo.vcxproj]
liblo.obj : error LNK2019: unresolved external symbol lo_message_get_argv referenced in function speed_liblo_read [D:\BuildAndGit\zynaddsubfx\rtosc\build\liblo.vcxproj]liblo.obj : error LNK2019: unresolved external symbol lo_message_serialise referenced in function speed_liblo_write 
[D:\BuildAndGit\zynaddsubfx\rtosc\build\liblo.vcxproj]
liblo.obj : error LNK2019: unresolved external symbol lo_message_deserialise referenced in function speed_liblo_read [D:\BuildAndGit\zynaddsubfx\rtosc\build\liblo.vcxproj]
liblo.obj : error LNK2019: unresolved external symbol lo_bundle_new referenced in function main [D:\BuildAndGit\zynaddsubfx\rtosc\build\liblo.vcxproj]
liblo.obj : error LNK2019: unresolved external symbol lo_bundle_add_message referenced in function main [D:\BuildAndGit\zynaddsubfx\rtosc\build\liblo.vcxproj]
liblo.obj : error LNK2019: unresolved external symbol lo_bundle_serialise referenced in function main [D:\BuildAndGit\zynaddsubfx\rtosc\build\liblo.vcxproj]
liblo.obj : error LNK2019: unresolved external symbol lo_bundle_free referenced in function main [D:\BuildAndGit\zynaddsubfx\rtosc\build\liblo.vcxproj]
D:\BuildAndGit\zynaddsubfx\rtosc\build\Debug\liblo.exe : fatal error LNK1120: 12 unresolved externals [D:\BuildAndGit\zynaddsubfx\rtosc\build\liblo.vcxproj]
</details>

See this line to be specific : `LINK : warning LNK4044: unrecognized option '/LC:/vcpkg/installed/x64-windows/debug/lib'; ignored [D:\BuildAndGit\zynaddsubfx\rtosc\build\liblo.vcxproj]`. This line is what causes the error.
MSVC uses `/LIBPATH` instead of `-L` and this seems to be defined somewhere in liblo (or anywhere else. Didn't find a definition within rtosc)

Forgive me if this issue is off topic. Don't know where to ask.

pinging @JohannesLorenz as he is the one who's working on the port.

**Comment by JohannesLorenz** [comment]
I got a lot more far. I removed all pkg_config stuff for Windows, instead used find_package, and also used a vcproj.json in the main folder. Now, I get:

```
LINK : fatal error LNK1104: cannot open file 'vcpkg_installed\x64-windows\lib\liblo.dll' [D:\a\rtosc\rtosc\liblo.vcxproj]
```

There is only a ".lib" file, but no ".dll" file in there.

For a minimal example, see https://github.com/JohannesLorenz/liblo-test/ , branch main.

**Comment by Rossmaxx** [comment]
We discussed this on discord and found that this issue has been fixed on master but not in 0.30, which is in vcpkg.

**Comment by Rossmaxx** [comment]
Closing as this is unrelated. The issue we had, fixed on our side and the pending issues can be fixed once we get a version bump.
